spec/models/project_spec.rb

```diff
@@ -83,7 +83,7 @@
   end
 
   describe '#related_projects' do
-    let(:organization) { Organization.create!(name: 'CodeMontage') }
+    let(:organization) { Organization.create!(name: 'CodeMontage', github_org: 'codeMontage') }
```

Comment:
Line is too long. [95/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.